### DISCUSSION TITLE
"name:nonlatin" missing in a number of layers

### DISCUSSION THREAD

**Discussion by aaime** [onthegomap/planetiler#1051] [General]
Hi,
I'm trying to use the planetiler output along with OpenMapTiles styles like [osm-bright-gl-style](https://github.com/openmaptiles/osm-bright-gl-style/blob/master/style.json).

The style often tries to render labels using the following expression:

        "text-field": "{name:latin} {name:nonlatin}",

Now... using the defaults, name:nonlatin seems to be present in a few layers:
* mountain_peak
* poi
* transportation_name

But it's also missing in the following:
* waterway
* water_name
* aerodrome_label
* place

The output is otherwise compatible with osm-bright-gl-style. I've searched around but could not find a simple tweak I can do to include it (other than defining a custom schema, maybe). Before I start digging into the code, did I miss anything obvious?

**Comment by aaime** [comment]
Hum... I guess I've found my answer:
https://github.com/openmaptiles/planetiler-openmaptiles

**Reply by msbarry** [reply]
Yeah the default openmaptiles profile tries to match https://github.com/openmaptiles/openmaptiles as close as possible which doesn't do the name logic on all layers. I'm not exactly sure why, maybe size? You could either open an issue there then planetiler will get it when they port over changes to it, or add just to planetiler-openmaptiles and hide it behind an opt-in flag.

**Reply by aaime** [reply]
Actually planetiler-openmaptiles is adding all the "nonlatin" fields I was missing, it's plain-jane planetiler that is missing them.
Probably a misunderstanding of mine.

**Reply by msbarry** [reply]
Planetiler embeds planetiler-openmaptiles repo so the output should be identical. If you can find a specific example feature that doesn't match please share here so I can understand the discrepancy further.
